packages/styles/.stylelintrc

```diff
@@ -7,6 +7,6 @@
     "selector-max-id": 1,
     "selector-no-qualifying-type": [true, { "ignore": ["attribute", "class"] }],
     "function-url-quotes": null,
-    "scss/dollar-variable-colon-space-after": "always-single-line"
+    "scss/dollar-variable-colon-space-after": null
```

Comment:
@leagrdv can you please add a comment in the code as well so that we know why this rule is deactivated?

Reply:
True... then you're right the PR is a good place to add the comment!